### PR TITLE
Remove booster dependency

### PR DESCRIPTION
**Pull request by ericonr**

### CODE REVIEW

**Comment by ericonr** [comment]
@henriquesimoes
@gustavosr8

File: util/decoders.cc [hunk 1]
```diff
@@ -1,20 +1,34 @@
...
+    template <typename T, typename U>
+    std::size_t operator()(const std::pair<T, U> &x) const
+    {
+        std::hash<T> hasher_t;
+        std::hash<U> hasher_u;
+
+        auto hash_t = hasher_t(x.first);
+        auto hash_u = hasher_u(x.second);
+
+        hash_u ^= hash_t + 0x9e3779b9 + (hash_u<<6) + (hash_u>>2);
```
Comment (gustavosr8) [inline util/decoders.cc:21]: Where did this `0x9e3779b9` value comes from?

File: util/decoders.cc [hunk 2]
```diff
@@ -1,20 +1,34 @@
...
+
+        return hash_t;
```
Comment (gustavosr8) [inline util/decoders.cc:23]: Shouldn't it be `hash_u` to be returned?

**Review by gustavosr8 — CHANGES_REQUESTED** [review]

File: util/decoders.cc (hunk 1 above)
Comment (ericonr) [reply util/decoders.cc:21]: It's the magic value used by boost themselves: https://stackoverflow.com/questions/35985960/c-why-is-boosthash-combine-the-best-way-to-combine-hash-values

File: util/decoders.cc (hunk 2 above)
Comment (ericonr) [reply util/decoders.cc:23]: Indeed!!
Thanks :)

I had changed it in the calculation, but forgot about the result

File: util/decoders.cc (hunk 1 above)
Comment (gustavosr8) [reply util/decoders.cc:21]: Do you mind to mention it in the commit message? Magic numbers can easily become the reason for a misunderstanding or doubt, IMHO.

Comment (ericonr) [reply util/decoders.cc:21]: Done >:)

**Review by henriquesimoes — APPROVED** [review]

**Pull request merged**